cmd/kubeadm/app/util/loadkernelmodule.go

```diff
@@ -0,0 +1,51 @@
+package util
```

Comment:
Why are we doing this in code? I thought the whole point was to move the kernel module loading logic into startup scripts. Doesn't kubeadm have startup scripts as well?

Reply:
just WIP. Will fix for moment, thanks

Reply:
Actuall, kubeadm itself don't have startup scripts. You can do this work in kubernetes-anywhere, but that's just one use case of kubeadm.